modules/ivc_champva/app/controllers/ivc_champva/v1/pega_controller.rb

```diff
@@ -43,6 +45,7 @@ def update_data(form_uuid, file_names, status, case_id)
           # We only need the first form, outside of the file_names field, the data is the same.
           form = ivc_forms.first
           send_email(form_uuid, ivc_forms.first) if form.email.present?
+          monitor.track_update_status(form_uuid, status)
```

Comment:
Log when the Pega status is updated (include `form_uuid` so that we can create a DD plot tracking latency between form creation and Status Update.